client/test/router_store.spec.js

```diff
@@ -734,15 +732,15 @@ describe('RouterStore', () => {
 
     describe('getURL', () => {
 
-      afterEach(() => {
-        resetAccountFixtures()
+      before(() => {
+        resetStore();
+        // Dispatcher.dispatch({ type: 'FORCE STORES TO GET INITIAL STATE' });
       });
 
-      // Skipped after account redux migration
-      // TODO: make the acount fixture as Immutable
-      describe.skip('defaultView', () => {
-        it('Should goto `AccountNew` (no account found)', () => {
-          let url = RouterStore.getURL().replace('#', '');
+      describe('defaultView', () => {
+        //ROMAINEDIT : this is not handled at the getter level
```

Comment:
Should't you test `Route.makeURL` too or instead? (and all other Router methods?)
This test is about how Router is handling redirection, shouldn't it be tested?